Core/Database.php

```diff
@@ -19,15 +15,6 @@ class Database
 
     public function __construct()
     {
-
-        // Carregar variáveis do .env
-        $dotenv = Dotenv::createImmutable(__DIR__ . '/../');
-        $dotenv->load();
```

Comment:
Esse `$dotenv->load()` ele consome o arquivo e disponibiliza na variável global `$_ENV`, depois disso não é necessário chamar outra vez. O ideal é sempre carregar na raiz do projeto, ou seja, no início para que todas as dependências tem acesso as envs.